File: Backend/Backend/DTOs/UserWorkLogDto.cs
namespace Backend.DTOs
{
    public class UserWorkLogDto
    {
        public double WorkTimeInMins { get; set; }
        public AppUserDto AppUser { get; set; }
        public TaskItemDto TaskItem { get; set; }
        public DateTime LogDate { get; set; }
    }
}

File: Frontend/Frontend/src/app/_dtos/userWorkLogDto.ts
import { AppUserDto } from "./appUserDto"
import { TaskItemDto } from "./taskItemDto"

export class UserWorkLogDto{
    appUser: AppUserDto = new AppUserDto()
    taskItem: TaskItemDto = new TaskItemDto()
    workTimeInMins: number = 0
    logDate: string = ''
}